src/state-summary/state-card-weather.html

```diff
@@ -0,0 +1,110 @@
+<link rel="import" href="../../bower_components/polymer/polymer.html">
+
+<link rel="import" href="../../bower_components/google-apis/google-legacy-loader.html">
+
+<dom-module id="state-card-weather">
+  <template>
```

Comment:
I also don't know if a chart is the best way to show the weather. We should put more emphasis on what is going on right now. I also think that probably instead of a `state-summary` we want to actually convert it to a stand alone card instead of showing it in line with the other data.

This example is obviously showing a lot of info but this is how we should approach showing the weather.

![screen shot 2016-12-05 at 6 21 16 pm](https://cloud.githubusercontent.com/assets/1444314/20910679/d8edaa88-bb17-11e6-939d-0b82e9153fe3.png)